src/viztracer/modules/vcompressor/vc_dump.c

```diff
@@ -240,6 +299,39 @@ load_events_from_file(FILE* fptr)
                 }
                 Py_DECREF(name);
                 break;
+            case VC_HEADER_INSTANT_EVENT:
+                loads_func = PyObject_GetAttrString(json_module, "loads");
+                if (!loads_func){
+                    goto clean_exit;
+                }
+                READ_DATA(&pid, uint64_t, fptr);
+                READ_DATA(&tid, uint64_t, fptr);
+                freadstrn(buffer, STRING_BUFFER_SIZE - 1, fptr);
+                name = PyUnicode_FromString(buffer);
+                freadstrn(buffer, STRING_BUFFER_SIZE - 1, fptr);
+                scope = PyUnicode_FromString(buffer);
+                READ_DATA(&count, uint64_t, fptr);
+                for (uint64_t i = 0; i < count / 2; i++) {
+                    READ_DATA(&ts, uint64_t, fptr);
+                    freadstrn(buffer, STRING_BUFFER_SIZE - 1, fptr);
```

Comment:
There's no bound for the argument string, so we should probably use `freadstr` declared above instead of `freadstrn`.